<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qucs-RFlayout</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pb_read">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_debug">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabelok</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_path_in">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>491</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_browse_in">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_input">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Input :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_format">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_format">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Format :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_path_out">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>491</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l_output">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Output :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_browse_out">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_write">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>470</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
